### ISSUE TITLE
[Camera Support]:  Reolink Duo Floodlight PoE

### ISSUE THREAD

**Issue by mclouse93** [blakeblackshear/frigate#7259]
### Describe the problem you are having

This camera does not allow you to change the encoding from H.265 to H.264. That being said, I did see that H.265 is supported. I verified that the URL I am using works by entering it in VLC media player and the stream came up as expected. Frigate is showing the "No frames have been received, check error logs" message. I can see the stream via the go2rtc dashboard and clicking on the "stream" option for the sub camera stream. If I select the main stream (the H.265 one) via the go2rtc dashboard, it is simply a black screen. 

I'm not sure why I'm receiving this message as I've configured this reolink camera similarly to the Amcrest cameras that I've installed. I've tried different URLs and I also tried a hwaccel_arg of c:v libx264 and also hevc_v4l2m2m. Neither solved anything for me. I shortened my config to the relevant pieces pertaining to the reolink camera. 

### Version

0.12.1-367d724

### Frigate config file

```yaml
go2rtc
  streams:    
    CarPort:
      - 'rtsp://admin:<omitted>@192.168.0.215:554//h265Preview_01_main'
    CarPort_sub:
      - 'rtsp://admin:<omitted>@192.168.0.215:554//h264Preview_01_sub'

 CarPort:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/Carport
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/CarPort_Sub
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      enabled: false
      width: 1536
      height: 576
    record:
      enabled: true
      expire_interval: 10
      retain:
        days: 7
        mode: all
      events:
        pre_capture: 5
        post_capture: 5
        retain:
          default: 10
          mode: motion
```

### Relevant log output

```shell
2023-07-23 10:56:57.727643699  [INFO] Starting Frigate...
2023-07-23 10:56:58.591701984  [2023-07-23 10:56:58] frigate.app                    INFO    : Starting Frigate (0.12.1-367d724)
2023-07-23 10:56:58.615278610  [2023-07-23 10:56:58] peewee_migrate                 INFO    : Starting migrations
2023-07-23 10:56:58.618913873  [2023-07-23 10:56:58] peewee_migrate                 INFO    : There is nothing to migrate
2023-07-23 10:56:58.635908152  [2023-07-23 10:56:58] frigate.app                    INFO    : Output process started: 291
2023-07-23 10:56:58.649507916  [2023-07-23 10:56:58] frigate.app                    INFO    : Camera processor started for Doorbell: 297
2023-07-23 10:56:58.662797210  [2023-07-23 10:56:58] frigate.app                    INFO    : Camera processor started for Porch: 298
2023-07-23 10:56:58.682564672  [2023-07-23 10:56:58] frigate.app                    INFO    : Camera processor started for CarPort: 302
2023-07-23 10:56:58.687605507  [2023-07-23 10:56:58] frigate.app                    INFO    : Capture process started for Doorbell: 304
2023-07-23 10:56:58.692296272  [2023-07-23 10:56:58] frigate.app                    INFO    : Capture process started for Porch: 305
2023-07-23 10:56:58.700059880  [2023-07-23 10:56:58] detector.coral_tpu             INFO    : Starting detection process: 289
2023-07-23 10:56:58.700063120  [2023-07-23 10:56:58] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as pci:0
2023-07-23 10:56:58.701171092  [2023-07-23 10:56:58] frigate.app                    INFO    : Capture process started for CarPort: 306
2023-07-23 10:56:58.708914980  [2023-07-23 10:56:58] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
2023-07-23 10:56:58.742764766  [2023-07-23 10:56:58] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:56:58.742920046  [2023-07-23 10:56:58] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:57:18.737276632  [2023-07-23 10:57:18] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:57:18.737438802  [2023-07-23 10:57:18] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:57:18.737721073  [2023-07-23 10:57:18] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x555a11a68c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:18.737726443  [2023-07-23 10:57:18] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:57:18.748044034  [2023-07-23 10:57:18] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x557c1297dd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:18.748641204  [2023-07-23 10:57:18] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:57:18.748821345  [2023-07-23 10:57:18] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:57:18.749286535  [2023-07-23 10:57:18] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:57:18.755273961  [2023-07-23 10:57:18] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:57:18.755759572  [2023-07-23 10:57:18] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:57:28.759805772  [2023-07-23 10:57:28] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:57:28.760162453  [2023-07-23 10:57:28] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:57:28.760167763  [2023-07-23 10:57:28] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x558b76a31c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:28.760171343  [2023-07-23 10:57:28] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:57:28.770310153  [2023-07-23 10:57:28] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x564875b91d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:28.775647409  [2023-07-23 10:57:28] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:57:28.775653439  [2023-07-23 10:57:28] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:57:28.775656349  [2023-07-23 10:57:28] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:57:28.777610341  [2023-07-23 10:57:28] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:57:28.778154182  [2023-07-23 10:57:28] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:57:38.783663504  [2023-07-23 10:57:38] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:57:38.783670184  [2023-07-23 10:57:38] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:57:38.783673454  [2023-07-23 10:57:38] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x559b70400c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:38.783676334  [2023-07-23 10:57:38] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:57:38.792709993  [2023-07-23 10:57:38] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x5580e7748d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:38.792716743  [2023-07-23 10:57:38] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:57:38.792720363  [2023-07-23 10:57:38] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:57:38.792723073  [2023-07-23 10:57:38] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:57:38.801053562  [2023-07-23 10:57:38] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:57:38.801705243  [2023-07-23 10:57:38] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:57:48.802655950  [2023-07-23 10:57:48] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:57:48.802803800  [2023-07-23 10:57:48] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:57:48.802944060  [2023-07-23 10:57:48] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x55f3ec62ec40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:48.807538705  [2023-07-23 10:57:48] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:57:48.813620722  [2023-07-23 10:57:48] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55a8f8d52d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:48.813627702  [2023-07-23 10:57:48] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:57:48.813630792  [2023-07-23 10:57:48] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:57:48.813633512  [2023-07-23 10:57:48] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:57:48.821209439  [2023-07-23 10:57:48] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:57:48.821511760  [2023-07-23 10:57:48] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:57:58.824611619  [2023-07-23 10:57:58] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:57:58.825241040  [2023-07-23 10:57:58] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:57:58.831305216  [2023-07-23 10:57:58] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x5646c7984c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:58.832140407  [2023-07-23 10:57:58] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:57:58.842368248  [2023-07-23 10:57:58] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55921921cd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:57:58.843209439  [2023-07-23 10:57:58] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:57:58.843338329  [2023-07-23 10:57:58] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:57:58.843482459  [2023-07-23 10:57:58] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:57:58.844613161  [2023-07-23 10:57:58] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:57:58.844861911  [2023-07-23 10:57:58] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:08.848423690  [2023-07-23 10:58:08] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:08.848843171  [2023-07-23 10:58:08] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:08.848848431  [2023-07-23 10:58:08] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x55ee7604cc40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:08.853152906  [2023-07-23 10:58:08] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:08.859431172  [2023-07-23 10:58:08] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55dac0875d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:08.859561562  [2023-07-23 10:58:08] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:08.859668632  [2023-07-23 10:58:08] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:08.859757792  [2023-07-23 10:58:08] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:08.866827180  [2023-07-23 10:58:08] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:08.867274561  [2023-07-23 10:58:08] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:18.869609909  [2023-07-23 10:58:18] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:18.869826909  [2023-07-23 10:58:18] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:18.869831929  [2023-07-23 10:58:18] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x56094acc6c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:18.870035619  [2023-07-23 10:58:18] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:18.885274986  [2023-07-23 10:58:18] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55c23d9edd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:18.885536286  [2023-07-23 10:58:18] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:18.885703096  [2023-07-23 10:58:18] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:18.885823716  [2023-07-23 10:58:18] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:18.888801059  [2023-07-23 10:58:18] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:18.889342580  [2023-07-23 10:58:18] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:28.890586037  [2023-07-23 10:58:28] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:28.890944078  [2023-07-23 10:58:28] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:28.890949318  [2023-07-23 10:58:28] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x5627d1b6bc40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:28.890952718  [2023-07-23 10:58:28] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:28.901395539  [2023-07-23 10:58:28] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55ae21a6dd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:28.901401849  [2023-07-23 10:58:28] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:28.901404729  [2023-07-23 10:58:28] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:28.901407179  [2023-07-23 10:58:28] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:28.908219546  [2023-07-23 10:58:28] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:28.908532876  [2023-07-23 10:58:28] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:38.911132215  [2023-07-23 10:58:38] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:38.911316775  [2023-07-23 10:58:38] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:38.911374145  [2023-07-23 10:58:38] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x55af1bba6c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:38.911521626  [2023-07-23 10:58:38] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:38.922535728  [2023-07-23 10:58:38] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x5635e2450d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:38.923153138  [2023-07-23 10:58:38] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:38.923748589  [2023-07-23 10:58:38] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:38.924270840  [2023-07-23 10:58:38] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:38.932071668  [2023-07-23 10:58:38] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:38.932257588  [2023-07-23 10:58:38] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:48.938059060  [2023-07-23 10:58:48] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:48.938220780  [2023-07-23 10:58:48] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:48.938336880  [2023-07-23 10:58:48] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x56108875bc40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:48.938431700  [2023-07-23 10:58:48] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:48.943749786  [2023-07-23 10:58:48] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55f8d5f36d80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:48.944117557  [2023-07-23 10:58:48] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:48.944122837  [2023-07-23 10:58:48] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:48.944126237  [2023-07-23 10:58:48] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:48.950815523  [2023-07-23 10:58:48] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:48.951104384  [2023-07-23 10:58:48] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:58:58.953838533  [2023-07-23 10:58:58] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:58:58.954164653  [2023-07-23 10:58:58] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:58:58.954390874  [2023-07-23 10:58:58] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x5577896b8c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:58.954583364  [2023-07-23 10:58:58] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:58:58.964086544  [2023-07-23 10:58:58] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x560288eadd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:58:58.964858035  [2023-07-23 10:58:58] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:58:58.964898735  [2023-07-23 10:58:58] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:58:58.964903455  [2023-07-23 10:58:58] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:58:58.971693042  [2023-07-23 10:58:58] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:58:58.972187722  [2023-07-23 10:58:58] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:59:08.975783752  [2023-07-23 10:59:08] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:59:08.976030862  [2023-07-23 10:59:08] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:59:08.976035732  [2023-07-23 10:59:08] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x5575bde8ec40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:59:08.976151203  [2023-07-23 10:59:08] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:59:08.991758140  [2023-07-23 10:59:08] ffmpeg.CarPort.record          ERROR   : [rtsp @ 0x55700837fd80] method DESCRIBE failed: 404 Not Found
2023-07-23 10:59:08.992346320  [2023-07-23 10:59:08] ffmpeg.CarPort.record          ERROR   : rtsp://127.0.0.1:8554/Carport: Server returned 404 Not Found
2023-07-23 10:59:08.992351450  [2023-07-23 10:59:08] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:59:08.992354570  [2023-07-23 10:59:08] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:59:08.993843082  [2023-07-23 10:59:08] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:59:08.993849182  [2023-07-23 10:59:08] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:59:18.996314850  [2023-07-23 10:59:18] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:59:18.996463300  [2023-07-23 10:59:18] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:59:18.996634710  [2023-07-23 10:59:18] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x55af21d92c40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:59:18.996776760  [2023-07-23 10:59:18] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:59:19.018574294  [2023-07-23 10:59:19] watchdog.CarPort               ERROR   : No new recording segments were created for CarPort in the last 120s. restarting the ffmpeg record process...
2023-07-23 10:59:19.018580174  [2023-07-23 10:59:19] watchdog.CarPort               INFO    : Terminating the existing ffmpeg process...
2023-07-23 10:59:19.018583424  [2023-07-23 10:59:19] watchdog.CarPort               INFO    : Waiting for ffmpeg to exit gracefully...
2023-07-23 10:59:19.024993781  [2023-07-23 10:59:19] frigate.video                  ERROR   : CarPort: Unable to read frames from ffmpeg process.
2023-07-23 10:59:19.025259321  [2023-07-23 10:59:19] frigate.video                  ERROR   : CarPort: ffmpeg process is not running. exiting capture thread...
2023-07-23 10:59:29.030831493  [2023-07-23 10:59:29] watchdog.CarPort               ERROR   : Ffmpeg process crashed unexpectedly for CarPort.
2023-07-23 10:59:29.031013703  [2023-07-23 10:59:29] watchdog.CarPort               ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-07-23 10:59:29.031161374  [2023-07-23 10:59:29] ffmpeg.CarPort.detect          ERROR   : [rtsp @ 0x55a7dc13ec40] method DESCRIBE failed: 404 Not Found
2023-07-23 10:59:29.031307354  [2023-07-23 10:59:29] ffmpeg.CarPort.detect          ERROR   : rtsp://127.0.0.1:8554/CarPort_Sub: Server returned 404 Not Found
2023-07-23 10:59:29.045624409  [2023-07-23 10:59:29] watchdog.CarPort               ERROR   : No new recording segments were created for CarPort in the last 120s. restarting the ffmpeg record process...
```

### FFprobe output from your camera

```shell
App would not open on my pc to run this.
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

 Reolink Duo Floodlight PoE

### Any other information that may be helpful

I run OpenMediaVault and run frigate as a container. I have 2 other cameras working properly, both are amcrest.

**Comment by NickM-27** [comment]
The RPi does not support hevc decoding with ffmpeg, you will not be able to use hwaccel with the main stream. Shouldn't be a problem if you use the sub stream though.

There is an error in the rtsp urls, you have `rtsp://admin:<omitted>@192.168.0.215:554//h265Preview_01_main` you should remove one of the `/` before `hXXXPreview...`

**Comment by YBonline** [comment]
I bought 1 of these cameras. First, make sure the firmware on the camera is up to date from their website, or contact support for a newer beta firmware. After the firmware is up to date, try connecting with go2rtc/Frigate ONLY, this camera gets overwhelmed with too many connections easily. Second, RTSP is extremely buggy on this camera. You can try leaving VLC open playing the stream for many hours and see if it crashes, my experience is it will. You can try and use the http-flv URL instead of RTSP and it works a LOT better, but I don't believe the main stream supports that, only the substream:
http://reolink_ip/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=username&password=password

If you want to avoid lots of headaches, although its a bit more money, get the Hikvision  DT387G2P-LSU/SL for a panoramic camera, been very reliable for me, and now I got 6 of them.

**Comment by mclouse93** [comment]
@NickM-27 removing one of the / did not fix it. :/ I'm also just running a normal desktop computer as a server, not an RPi (not sure if that matters, just mentioning).

@YBonline I had also read that it was buggy, but figured I'd chance it. Was not expecting to get these kinds of headaches though since I read about the new firmware that was to fix a lot of the RTSP issues. I did verify that I do have the latest firmware, but maybe I'll reach out about the beta firmware to see what they have to offer. The ONLY reason I picked this camera was that it had the floodlights built into it. My wife wanted a camera with floodlights that way when she gets home, the lights will turn on and she'll feel more safe. So the panoramic view wasn't the selling point for me. If you know of any other cameras with floodlights built in, I'd love to see it! It doesn't appear that the Hikvision you mentioned has floodlights.
That being said, I believe I could just purchase a separate floodlight and make an automation in Home Assistant to trigger it on based on the detection from a different camera.

**Comment by NickM-27** [comment]
> removing one of the / did not fix it. :/ I'm also just running a normal desktop computer as a server, not an RPi (not sure if that matters, just mentioning).

For now I would suggest configuring frigate to connect directly to the camera and without hwaccel so we can see if the most basic config works

> It doesn't appear that the Hikvision you mentioned has floodlights.

it has LEDs and you don't need floodlights when you have an actually good image sensor (like the hikvision has and the reolink doesn't).

**Comment by mclouse93** [comment]
@NickM-27 Okay, the hwaccel part is removed. No change. When you say to configure it directly to the camera, do you mean to also remove the go2rtc stream portion as well and put the link in the path under the cameras portion?
The floodlights are to light up the driveway for my wife when she gets home at night. There are nights when she doesn't get home until 11 pm and she doesn't like walking up the driveway in the dark.

Thanks for the quick responses on this!!

**Comment by NickM-27** [comment]
> Okay, the hwaccel part is removed. No change. When you say to configure it directly to the camera, do you mean to also remove the go2rtc stream portion as well and put the link in the path under the cameras portion?

There is no need to remove the go2rtc section just simply update the `cameras` section to use the cameras URL instead of the restream

> The floodlights are to light up the driveway for my wife when she gets home at night. There are nights when she doesn't get home until 11 pm and she doesn't like walking up the driveway in the dark.

Right, like I said the camera has strong LED lights

**Comment by mclouse93** [comment]
@NickM-27 oohhh. Misunderstood on the LEDs somehow. lol My apologies. 

Interestingly enough, using the rtsp url at least gives a snapshot on frigate and removes all of the errors. Now when I click on the camera feed, it just doesn't load. I suppose this is probably part of the RTSP bugginess that I was hoping was fixed with the newer firmware. When I go to the go2rtc dashboard and click on the main feed (H.265) it still appears as black.

**Comment by mclouse93** [comment]
Correction, changing the mode from MSE to JSMPEG has seemed to fix the issue as far as viewing the feed in frigate.

We shall see how well this works! Thanks!

**Comment by NickM-27** [comment]
You can try using `preset-vaapi` for hwaccel

**Comment by mclouse93** [comment]
Would that be a global command or specific to this reolink camera?

**Comment by NickM-27** [comment]
should work globally for all cameras

**Comment by mannypenny843** [comment]
Just an FYI, Reolink released an update (v3.0.0.5586_2601148691) for the Duo Floodlight PoE that fixes the main RTMP stream, which was previously not functional, and it's the most stable stream I've ever seen from this camera.
